core/templates/pages/admin-page/roles-tab/admin-roles-tab.component.html

```diff
@@ -92,14 +92,14 @@ <h2>Role editor</h2>
           <mat-icon aria-label="Example icon-button with a heart icon">edit</mat-icon>
         </button>
       </div>
-      <div *ngIf="!username">Please fillout the username above to editor the user role.</div>
+      <div *ngIf="!username">Please fillout the username above to edit the user role.</div>
```

Comment:
Still not fixed. Start with "Please fill in..."

Reply:
From last comment I thought you were asking just to change editor to edit! Changed `fillout` to `fill in`